### PR TITLE
Provider 416 revisar deprecation warning

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [ ] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: microservices/realtime/config/packages/security.yaml [hunk 1]
```diff
@@ -7,7 +8,7 @@ security:
             pattern: ^/(_(profiler|wdt)|css|images|js)/
             security: false
         main:
-            anonymous: true
+#            anonymous: true
```
Comment (mmadariaga) [inline microservices/realtime/config/packages/security.yaml:11]: Remove this line or set it to false

File: web/rest/client/config/packages/framework.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 framework:
     secret: '%env(APP_SECRET)%'
+    router: {utf8: true }
```
Comment (mmadariaga) [inline web/rest/client/config/packages/framework.yaml:3]: use same sintax as above

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**